pom.xml

```diff
@@ -243,6 +243,12 @@
         <!-- Carbon kernel version -->
         <carbon.kernel.registry.imp.pkg.version>[1.0.1, 2.0.0)</carbon.kernel.registry.imp.pkg.version>
         <carbon.kernel.version>4.4.7</carbon.kernel.version>
+        <carbon.kernel.package.import.version.range>[4.4.0, 5.0.0)</carbon.kernel.package.import.version.range>
+
+        <!--Carbon identity version-->
+        <identity.framework.version>${project.version}</identity.framework.version>
+        <carbon.identity.package.export.version>${project.version}</carbon.identity.package.export.version>
```

Comment:
Is there any usage of this property ? If something needs to be exported from this project ideally it should be something like identity.application.authz.xacml.export.version. I think we can remove this property